### ISSUE TITLE
Please do not use the "CoC" acronym to refer to this document

### ISSUE THREAD

**Issue by vivienlacourba** [w3c/PWETF#368]
The acronym "CoC" is particularly badly chosen even more in the context of a Code of Conduct.

Also we have already identified that W3C uses its own jargon with a very heavy use of acronyms which is making on-boarding newcomers pretty hard. So I would recommend to avoid using yet another acronym to refer to this document and simply call it the "Code of Conduct".

cc: @koalie as I already relayed that feedback to the W3C Communications Team.

**Comment by vivienlacourba** [comment]
FYI "CoC" is used 13 times in: https://www.w3.org/policies/code-of-conduct/drafts/20240202/

**Comment by chrisn** [comment]
I agree. Also, I don't think the abbreviation particularly helps with readability in the document itself, so would suggest updating it to write Code of Conduct in full.

**Comment by dbooth-boston** [comment]
I don't remember why it didn't become "Code of Professional Conduct" when the word "Ethics" was dropped from the title, but that would solve the problem.

**Comment by TzviyaSiegman** [comment]
Thanks for the input. We changed the name to Code of Conduct (from Code of Ethics and Professional Conduct) because it's the common and accepted term. Ethics is not really discussed. We replaced all instances of the acronym "CEPC" with "CoC". If that is confusing we can find a better way to word it.

**Comment by koalie** [comment]
> We replaced all instances of the acronym "CEPC" with "CoC".

May I suggest that instead, you use "(C|c)ode"?

I was editor and chair of the original Positive Work Environment group and we'd made sure to avoid "Code of conduct"/"Coc" then. I'm sympathetic to @vivienlacourba's point!

**Comment by TzviyaSiegman** [comment]
Would it be okay to spell out "code of conduct"?

**Comment by dbooth-boston** [comment]
That would solve the immediate problem, but if we want to prevent future unfortunate abbreviations of the title -- including in other writings that we don't control -- it may be wise to change the title itself, for example to "Code of Professional Conduct".

**Comment by cwilso** [comment]
We had this discussion at great length in the CG.  There was a strong feeling that the common name for this thing was "Code of Conduct", and we should use that name; with a minority bu strong opinion that we had investment at W3C in the "CEPC" name.  I don't think we should consider other names.

The common way to refer to this would be to call it "the Code".

**Comment by koalie** [comment]
> Would it be okay to spell out "code of conduct"?

-1 from me because people will abbreviate (and if they're like me they'll read the abbreviated version in their head as well). 
-1 to Vivien's concern as well because while "code" is indeed generic, I think the chances of ambiguity are negligible given the context the word will be used in.

Given that the CG resolved to call this "Code of Conduct" and not "Code of Professional Conduct" for example, and given that its acronym is unfortunate given the subjects the document deals with, I think that barring renaming this "Conduct Code" and abbreviating "CC" (which also is known in the community to exist for "creative commons"), then the second best trade-off is using "Code" in the body.
